Core/include/Acts/Geometry/GeometryHierarchyMap.hpp

```diff
@@ -208,43 +213,44 @@ inline GeometryHierarchyMap<value_t>::GeometryHierarchyMap(
           std::vector<InputElement>(elements.begin(), elements.end())) {}
 
 template <typename value_t>
-template <typename iterator_t>
 inline void GeometryHierarchyMap<value_t>::sortAndCheckDuplicates(
-    iterator_t beg, iterator_t end) {
+    std::vector<InputElement>& elements) {
```

Comment:
Hmm are we sure we want to shrink the interface from a general iterator interface to enforcing a `std::vector`?
If change this, why not accept a range in the sense of a C++20 range?

Reply:
Ah I didn't see stephens comment